apps/site-app/src/containers/StoreOnline/Google/StoreGoogle.tsx

```diff
@@ -107,9 +107,11 @@ export const StoreGoogle = (props: MyProps) => {
     ? <Checkbox toggle disabled={disabled} onClick={onConnectClick} />
     : <ButtonSecondary onClick={onConnectClick} disabled={disabled} loading={loading}><FormattedMessage {...message } /></ButtonSecondary>;
 
+  const linkConnect = props.children ? <a onClick={onConnectClick}>{props.children}</a> : undefined;
+
   return (
     <div>
-      {connectVia}
+      {linkConnect ?? connectVia}
```

Comment:
We either have a button, a toggle or something else. Might need to clean that one day.